LayoutTests/ipc/read-pixels-buffer-object-overflow.html

```diff
@@ -0,0 +1,90 @@
+            });
+            const remoteRenderingBackend = streamConnection.newInterface("RemoteRenderingBackend", renderingBackendIdentifier);
+
+            const didInitializeReply = streamConnection.connection.waitForMessage(renderingBackendIdentifier, IPC.messages.RemoteRenderingBackendProxy_DidInitialize.name, 1)
+            streamConnection.connection.setSemaphores(didInitializeReply[0].value, didInitializeReply[1].value);
+
+            const graphicsContextGLIdentifier = Math.floor(Math.random() * 0x1000000);
+            const streamConnection2 = CoreIPC.newStreamConnection();
+            CoreIPC.GPU.GPUConnectionToWebProcess.CreateGraphicsContextGL(0, {
+                identifier: graphicsContextGLIdentifier,
+                attributes: {
+                    alpha: true,
+                    depth: true,
+                    stencil: false,
+                    antialias: true,
+                    premultipliedAlpha: true,
+                    preserveDrawingBuffer: false,
+                    powerPreference: 0,
+                    isWebGL2: true,
+                    windowGPUID: 0,
+                    xrCompatible: false,
+                    failContextCreationForTesting: 0
+                },
+                renderingBackendIdentifier: renderingBackendIdentifier,
+                serverConnection: streamConnection2,
+            });
+            const remoteGraphicsContextGL = streamConnection2.newInterface("RemoteGraphicsContextGL", graphicsContextGLIdentifier);
+
+            const wasCreatedReply = streamConnection2.connection.waitForMessage(graphicsContextGLIdentifier, IPC.messages.RemoteGraphicsContextGLProxy_WasCreated.name, 1)
+            streamConnection2.connection.setSemaphores(didInitializeReply[0].value, didInitializeReply[1].value);
+
+            remoteGraphicsContextGL.ReadPixelsBufferObject({
+                arg0: {
+                    location: {
+                        x: 0,
+                        y: 0
+                    },
+                    size: {
+                        width: 0xffffffff,
+                        height: 0xffffffff
+                    }
+                },
+                format: 0,
+                type: 0,
+                offset: 0,
+                alignment: 0,
+                rowLength: 0,
+            });
+
+            streamConnection2.connection.waitForMessage(graphicsContextGLIdentifier, IPC.messages.RemoteGraphicsContextGLProxy_addDebugMessage.name, 5)
+        }
+
+        globalThis.testRunner?.waitUntilDone();
+
+    </script>
+    <script type="module">
+
+        import { CoreIPC, IPCWireTap } from './coreipc.js';
+
+        window.CoreIPC = CoreIPC;
+        window.IPCWireTap = IPCWireTap;
+
+        setTimeout(() => {
+            main().catch(e => {
+                console.log(e);
+            }).finally(() => {
+                globalThis.testRunner?.notifyDone();
+            });
+        }, 100);
```

Comment:
The test will crash future tests, since you don't cleanup the connections.
You can observe this by running the test with Debug build , run-webkit-tests --debug --repeat 1000.
It's better to not test this way.